core/test/transform_test.cpp

```diff
@@ -5,9 +5,9 @@
 
 #include "test_macros.h"
 
-#include "../operators.h"
 #include "make_sparse.h"
 #include "scipp/core/dimensions.h"
+#include "scipp/core/operators.h"
```

Comment:
Why did you have to move the header? Is it now required in any other public header files?

Reply:
This was left over from a previous attempt, it is not needed now so will revert these changes.